### PR TITLE
mobile version : portfolio navigation , welcome and introduction updated

### PR DESCRIPTION
**Pull request by PowerLevel9000**
well this is my mobile version project, i am working on mobile verison for the first time 

  - earlier i used to work on desktop versions and use 3media querries to for responisive in small medium and large screens
  -  this is my first time so suggestions are welcome
 In this project i try to stick with templated provided and added my personal details there also added some 
navigation bar interection 
social icons intrections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,51 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b89b7c8adf.js" crossorigin="anonymous"></script>
 </head>
 <body>
+
+  <main>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you change this to a `<header>` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

Websites usually have the hierarchy of `<header><main><footer>`
in the `header`, you have the `<nav>` element and title, in the `main` you have the rest of the page and in the `footer` you have a contact form or social media links or copyright information.

File: index.html [hunk 2]
```diff
@@ -5,9 +5,51 @@
...
+
+    <nav>
+      <div class="logo"><h2>AS Pathak</h2></div>
```
Comment (Nyame-Wolf) [inline index.html]: Kindly change this into a link tag `<a>`This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 3]
```diff
@@ -5,9 +5,51 @@
...
 <body>
+
+  <main>
+
+    <nav>
+      <div class="logo"><h2>AS Pathak</h2></div> 
+      <img class="menu" src="media\Union.png" alt="hamburger" />
+    </nav>
+
+    <div class="introduction">
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you change this from a `<div>` tag to a `<section>` tag. It represents a generic standalone section of the document [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.I

File: index.html [hunk 4]
```diff
@@ -5,9 +5,51 @@
...
 </head>
 <body>
+
+  <main>
+
+    <nav>
+      <div class="logo"><h2>AS Pathak</h2></div> 
+      <img class="menu" src="media\Union.png" alt="hamburger" />
+    </nav>
+
```
Comment (Nyame-Wolf) [inline index.html:21]: [OPTIONAL] this here should represent the end of the `header` tag. This is where you can include your `main` tag. The main will contain the different sections of your page until you get to the `footer`.

File: index.html [hunk 5]
```diff
@@ -5,9 +5,51 @@
...
+          <i class="fa-brands fa-instagram"></i>
+        </a>
+        <a href="https://www.facebook.com/profile.php?id=100010279544328" target="_blank">
+          <i class="fa-brands fa-facebook"></i>
+        </a>
+        <a href="https://youtube.com/@code_gamer2.068" target="_blank">
+          <i class="fa-brands fa-youtube"></i>
+        </a>
+         
+      </div>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you change this from a `div` tag to a list  that is a `ul` tag to group them, and `li` per icon link
It is important to use correct semantic tags to ensure that screen readers don't get confused when reading your page.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @PowerLevel9000  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 5 above)
Comment (PowerLevel9000) [reply index.html]: thanks for your overall suggestions i just have a slip of mind i know these things about header footer main section  footer
but compleltey forget about  its implimentation  in my project

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+  line-height: 20px;
+  color: var(--theame-color);
+  align-items: center;
+  justify-content: center;
+  margin-left: 10px;
+  text-decoration: none;
+}
+
+.logo:hover {
+  transform: scale(0.9);
```
Comment (jerryowusu) [inline styles.css:32]: [OPTIONAL] There is an empty blank space to the right side of your page which is not availble on the figma design. Kindly check and fix it to make it look like the given design. :pray: 

Your design

Screenshot from 2022-12-02 09-33-31

Figma design

Screenshot from 2022-12-02 08-24-48

**Review by jerryowusu — APPROVED** [review]
Hi @PowerLevel9000 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

clap approve

## Highlights
All linters are passing :heavy_check_mark: 
Used correct GitHub flow :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 6 above)
Comment (PowerLevel9000) [reply styles.css:32]: well i never use mobile responsive thing and as it look wierd when i put reponsive units insted of px on desktop mode 

so i didn't borther to use responsive unit such em rem and % well i devloped it for iphone 12

image

if you switch some setting there may be you will see my actual project

**Comment by PowerLevel9000** [comment]
Hi @jerryowusu 


I hope this message finds you well. I wanted to take a moment to address a concern I have regarding my previous message. Upon reflection, I realize that my tone may have been perceived as offensive or argumentative, and for that, I sincerely apologize. As was a junior, I am constantly striving to improve my professional skills, and I acknowledge that there is always room for growth. I want to assure you that I had no ill intentions in my communication, as the limitations of text make it challenging to convey emotions effectively. I also have a curious and talkative nature, which sometimes leads me to question why things should be done in a certain way. However, I understand that my approach may not always be appropriate, and I apologize for any discomfort caused.

In the same vein, I extend my apologies @Nyame-Wolf as well. I want to clarify that my intention was never to offend or engage in an argument. Rather, I sought to express my thoughts and gain a deeper understanding of the subject matter.

Thank you for your understanding and patience.

@PowerLevel9000

**Pull request merged**